iothub/device/src/InternalClient.cs

```diff
@@ -111,9 +110,7 @@ public InternalClient(IotHubConnectionString iotHubConnectionString, ITransportS
         {
             if (Logging.IsEnabled) Logging.Enter(this, transportSettings, pipelineBuilder, nameof(InternalClient) + "_ctor");
 
-#if NET451
-            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls;
-#endif
+            TlsVersions.SetLegacyAcceptableVersions();
```

Comment:
Does this take care of restricting in amqp/mqtt as well ?

Reply:
From my testing, it only seems to impact HTTP or MQTT/AMQP over websocket. Based on documentation and of the frameworks we support, it should impact .NET 4.5.1 only.

My next PR should address straight up MQTT.